### ISSUE TITLE
Notifications in the Global Nav

### ISSUE THREAD

**Issue by HeathMeyette** [Esri/global-nav#33]
@jonathantneal has there been any work to include notifications in the global nav (shown below)? Please advise.

![screen shot 2017-08-04 at 4 34 08 pm](https://user-images.githubusercontent.com/6730794/28990493-dc53336c-7932-11e7-84a4-1cf4fe9847a2.png)

**Comment by asangma** [comment]
cc @gschnall

**Comment by asangma** [comment]
cc @jonathantneal

**Comment by bstifle** [comment]
yes for shopping. this can be applied to anything in the nav i reckon.
<img width="523" alt="alerts" src="https://user-images.githubusercontent.com/14083530/28991435-53ce2844-793c-11e7-9e08-22bc3453c9d7.png">

**Comment by asangma** [comment]
@jonathantneal @bstifle 

To be clear, we will likely need this to retain one of the existing features of the nav in production on ArcGIS Online. 👈  True @gschnall ?

**Comment by HeathMeyette** [comment]
@bstifle @asangma We also need to think about Marketplace once they take on the global nav. They  may need both a shopping cart and notifications.

**Comment by bstifle** [comment]
designed as well @HeathMeyette see
http://proto.esri.com/#/detail/Navigation/Global%20Mobile%20Navigation/1
http://proto.esri.com/#/detail/Navigation/Global%20Navigation/1

**Comment by gschnall** [comment]
@asangma Yes, it looks like alerts will be similar to how we are integrating the app switcher. Although, I have no idea how the shopping cart works. I'd have to look into that.

**Comment by HeathMeyette** [comment]
@gschnall @bstifle @asangma I think we will need to get some alignment between teams here. Seems like the Marketplace and Esri.com folks need to be talking to discuss how these interact (or don't). In short, will this be a "global" esri shopping cart that includes both Marketplace items and esri.com related items? Or will they be exclusive?

**Comment by bstifle** [comment]
That sounds like a Cassidy question :)

**Comment by HeathMeyette** [comment]
@bstifle yes indeed it is.

**Comment by asangma** [comment]
I invoke, with all my feeble powers...@CassidyB!
